Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -943,6 +943,20 @@
       - ClothingOuterWinterSci
       - ClothingOuterWinterRobo
       - ClothingOuterWinterSec
+      - ClothingHeadHatCaptain
```

Comment:
you need to sort this close to the right jobs stuff so that the captain's beret is near all the captain's clothes.

Reply:
??? the winter coats and the mantles aren't near the jobs stuff, but i can do that, yeah.

Reply:
done